docs/contributors/release.md

```diff
@@ -253,6 +253,128 @@ Now, the branch is ready to be used to publish the npm packages.
 
 Now, the npm packages should be ready and a patch can be created and committed into the corresponding WordPress SVN branch.
 
+
+### Standalone Package Releases 
+
+The following workflow is needed when packages require bug fixes or security releases to be published to _npm_ outside of a WordPress release cycle.
+
+Note: Both the `master` and `wp/trunk` branches are restricted and can only be _pushed_ to by the Gutenberg Core team.
+
+Identify the commit hashes from the pull requests that need to be ported from the repo `master` branch to `wp/trunk`
+
+The `wp/trunk` branch now needs to be prepared to release and publish the packages to _npm_.
+
+Open a terminal and perform the following steps:
+1. `git checkout master`
+2. `git pull`
+3. `git checkout wp/trunk`
+4. `git pull`
+
+Before porting commits check that the `wp/trunk` branch does not have any outstanding packages waiting to be published:
+1. `git checkout wp/trunk`
+2. `npm run publish:check`
+
+Now _cherry-pick_ the commits from `master` to `wp/trunk`, use `-m 1 commithash` if the commit was a pull request merge commit:
+1. `git cherry-pick -m 1 cb150a2`
+2. `git push` 
+
+Whilst waiting for the Travis CI build for  `wp/trunk` [branch to pass](https://travis-ci.com/WordPress/gutenberg/branches) identify and begin updating the `CHANGELOG.md` files:
+1. `git checkout wp/trunk`
+2. `npm run publish:check`
+> Example
+> ```shell
+> npm run publish:check
+> @wordpress/e2e-tests
+> @wordpress/jest-preset-default
+> @wordpress/scripts
+> lerna success found 3 packages ready to publish
+> ```
+
+Check the versions listed in the current `CHANGELOG.md` file, looking through the commit history of a package e.g [@wordpress/scripts](https://github.com/WordPress/gutenberg/commits/master/packages/scripts) and look out for _"chore(release): publish"_ and _"Update changelogs"_ commits to determine recent version bumps, then looking at the commits since the most recent release should aid with discovering what changes have occurred since the last release.
```

Comment:
In theory, PRs should cover that part, but in reality, there is only a handful of people who do it. I guess the section about updating changelogs and picking the proper version bump is common for all types of npm releases.

Reply:
True, semantic-release, there's a Lerna semantic release even, but getting to that stage whilst still making it easy for contributors is the hard part.

We could start enforcing (via GitHub) actions that changelogs are updated, but again, that's not the easiest for contributors (its hard and time consuming for experienced contributors)